docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -92,11 +92,66 @@ to pass the new file with `--kubeconfig` on the command lines when
 using kcp or KubeStellar.
 
 
+### Space Manager
+
+The Space Manager will use space provider to create and manage underlying cluster
```

Comment:
but the underlying thing is not necessarily a _cluster_, right?

Reply:
That's correct.  In the case of kubeflex for example they are really a control plane or an api server.  I guess the question is how do we define 'space'.  And the answer perhaps is that a space, similar to a kcp workspace that it replaces, provides a mechanism for multi cluster and multi tenancy by providing isolated api resources and endpoint?